examples/step-51/doc/intro.dox

```diff
@@ -55,11 +54,22 @@ element-level solutions.  This is known as the
 solution process.
 </ol>
 
-The above procedure also has a linear algebra interpretation and is referred to
-as static condensation. Let us write the complete linear system associated to
-the HDG problem as a block system with the discrete DG (cell interior)
-variables $U$ as first block and the skeleton (face) variables $\Lambda$ as the
-second block:
+<h4> Relation with Static Condensation </h4>
+The above procedure also has a linear algebra interpretation---referred to
+as <i>static condensation</i>---that was exploited to reduce the size of the
+ global linear system by Guyan in the context of continuous Finite Elements
+ @cite G65, and by Fraeijs de Veubeke for mixed methods @cite F65. In the latter
+ case (mixed formulation), the system reduction was achieved through the use of
+ discontinuous fluxes combined with the introduction of an additional auxiliary
+ <i>hybrid</i> variable that approximates the trace of the unknown at the
+ boundary of every element. This procedure became known as hybridization 
+and---by analogy---is the reason why the local discontinuous Galerkin method
```

Comment:
While you're here, editing this file again, can you remove the spaces that are at the beginning of some but not all of the lines above?

Reply:
With some luck I caught all the rouge spaces now.